lib/v2/oceanengine/arithmeticIndex.js

```diff
@@ -0,0 +1,141 @@
+const dayjs = require('dayjs');
+const { art } = require('@/utils/render');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+const path = require('path');
+const config = require('@/config').value;
+
+// Parameters
+const CACHE_MAX_AGE = config.cache.contentExpire * 6; // 6 hours
```

Comment:
```suggestion
const CACHE_MAX_AGE = config.cache.contentExpire;
```
Please use it directly.